Exiled.Events/Patches/Events/Map/GeneratorActivated.cs

```diff
@@ -49,7 +47,7 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             // if (!ev.IsAllowed)
             //   return;
             newInstructions.InsertRange(
-                index,
```

Comment:
idc about that, I'd like to simply rename the event and no to change patch
(if you can disable event by IsAllowed, then event is called *before* event)

Reply:
Bruh, bad naming. It's fired before generator activates and has IsAllowed property but called Activat**ed**

Reply:
Oh i didn't seen than it's isallowed bruh